hoodie-common/src/main/java/com/uber/hoodie/common/table/HoodieTableMetaClient.java

```diff
@@ -223,6 +248,13 @@ public synchronized HoodieActiveTimeline reloadActiveTimeline() {
     return activeTimeline;
   }
 
+  public ConsistencyGuardConfig getConsistencyGuardConfig() {
+    if (null == consistencyGuardConfig) {
```

Comment:
L80 above should ensure this is non-null?

Reply:
Fixed.